src/models/subgraph-entities/v1/distribute-to-ticket-mod-event.ts

```diff
@@ -1,54 +1,28 @@
 import { BigNumber } from '@ethersproject/bignumber'
+import { Project } from 'models/subgraph-entities/vX/project'
 import {
-  parseProjectJson,
-  Project,
-  ProjectJson,
-} from 'models/subgraph-entities/vX/project'
+  parseBigNumberKeyVals,
+  parseSubgraphEntitiesFromJson,
+} from 'utils/graph'
 
-import {
-  BaseEventEntity,
-  BaseEventEntityJson,
-  parseBaseEventEntityJson,
-} from '../base/base-event-entity'
-import {
-  parsePrintReservesEventJson,
-  PrintReservesEvent,
-  PrintReservesEventJson,
-} from './print-reserves-event'
+import { Json } from '../../json'
+import { BaseEventEntity } from '../base/base-event-entity'
+import { PrintReservesEvent } from './print-reserves-event'
 
 export interface DistributeToTicketModEvent extends BaseEventEntity {
-  project: Partial<Project>
```

Comment:
no more `Partial` in any subgraph models. it makes typing much more straightforward. and although objects like this will almost always be partials in practice, the type will be narrowed when the query is made.